installinstallmacos.py

```diff
@@ -511,8 +511,8 @@ def main():
         bad_dirs = ['Documents', 'Desktop', 'Downloads', 'Library']
         for bad_dir in bad_dirs:
             if bad_dir in current_dir:
-                sys.exit('This command cannot be run from within certain home directories, so please '
-                'run again from somewhere else, such as /Users/Shared')
+                print('Running this script from {} may not work as expected. If this does not run as expected, '
+                'please run again from somewhere else, such as /Users/Shared'.format(current_dir))
 
```

Comment:
print this warning to stderr, please

Reply:
Second line indentation looks wrong

Reply:
The script doesn't use .format()-style string interpolation anywhere else. Should stick to the established style.

Reply:
Also: what happens if it's run from `/Users/Shared/Documents`?

Reply:
It is fine in `/Users/Shared/Documents` as it won't pass the first check on line 510 (which is looking for user dir). So we check for user dir first, then get more specific after that.

```
 sudo ./installinstallmacos.py
Password:
Downloading https://swscan.apple.com/content/catalogs/others/index-10.15-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog...
...
```